terraform/modules/configs/deploy-all/main.tf

```diff
@@ -404,6 +423,15 @@ module "ecs_sidekiq" {
   resource_name_suffix = "SIDEKIQ"
 }
 
+module "ecs_s3_virus_scan" {
+  source               = "../../ecs"
+  environment          = var.environment
+  subnet_ids           = split(",", data.aws_ssm_parameter.public_web_subnet_ids.value)
```

Comment:
Does this service need to be in the public subnets group, as it is only called by the Lambda function?  If it makes requests out to the internet (I assume it does), these calls would route through the NAT Gateway.